web/src/app/schedules/ScheduleCreateDialog.js

```diff
@@ -17,36 +17,35 @@ const mutation = gql`
   }
 `
 
-export default class ScheduleCreateDialog extends React.PureComponent {
-  state = {
+export default function ScheduleCreateDialog(props) {
+  const [schedule, setSchedule] = useState({
```

Comment:
let's keep these names as `value` and `setValue`, as this would remain consistent with how we use forms across the app. Then, `name`, `description`, `timeZone`, and favorite can be listed flat for the object initialization.